docs/source/admin/traffic_vault.rst

```diff
@@ -33,6 +33,7 @@ In order to use the PostgreSQL backend for Traffic Vault, you will need to set t
 :password:                  The password to use when connecting to the database
 :port:                      The port number that the database listens for new connections on (NOTE: the PostgreSQL default is 5432)
 :user:                      The username to use when connecting to the database
+:encryption_key_location:   The location on-disk for the key used to encrypt secrets before they are stored.
```

Comment:
instead of calling this `encryption_key_location`, which might be confused w/ a public key, we should call it `aes_key_location` and note that it must be an AES key which provides both encryption and decryption